spec/provider/hephy_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'spec_helper'
+require 'dpl/provider/hephy'
+
+describe DPL::Provider::Deis do
```

Comment:
this should describe `DPL::Provider::Hephy`

That seems to be the source of CI failure

Reply:
Thank you @kingdonb . Fixed with latest commit rewrite. Let's see if CI will pass.